File: kotlin-pr/50/kotlin/code-quality-naming/UserFilter.kt
class UserFilter {
    // {fact rule=code-quality-naming@v1.0 defects=0}
    private var userCredentials: MutableMap<String, String>? = null

    fun storeUserCredentials(username: String, password: String): Boolean {
        if (userCredentials == null) {
            userCredentials = mutableMapOf()
        }

        val hashedPassword = username + password // Simplified hashing
        userCredentials!![username] = hashedPassword
        
        return userCredentials!!.containsKey(username)
    }

    fun getUserCredentials(username: String): String? {
        return userCredentials?.get(username)
    }
    // {/fact}
}

File: scala-pr/54/scala/code-quality-naming/UserFilter.scala
import scala.collection.mutable

class UserFilter {
  // {fact rule=code-quality-naming@v1.0 defects=0}
  private var userCredentials: Option[mutable.Map[String, String]] = None

  def storeUserCredentials(username: String, password: String): Boolean = {
    if (userCredentials.isEmpty) {
      userCredentials = Some(mutable.Map.empty[String, String])
    }

    val hashedPassword = username + password // Simplified hashing
    userCredentials.get(username) = hashedPassword
    
    userCredentials.get.contains(username)
  }

  def getUserCredentials(username: String): Option[String] = {
    userCredentials.flatMap(_.get(username))
  }
  // {/fact}
}